### DISCUSSION TITLE
no se como hacer mas de un note book para mis kattas

### DISCUSSION THREAD

**Discussion by ING-ANGEL-GC** [LaunchX-InnovaccionVirtual/CursoIntroPython#92] [Q&A] [answered]
aun que se hacer las katas no se como hacer mas un note book para organizarlos alguien me ayuda

**Comment by nosoyhector** [comment]
En Visual Studio Code
**Ctrl**+**Shift**+**P**

Y pones **Jupyter: Create New Jupyter Notebook**
O con el simple hecho de salvar/crear tu archivo con la extensión **.ipynb**

**Reply by ING-ANGEL-GC** [reply]
caundo hago una hoja de codigo y la guardo como .ipynb se para lo que tengo guarado en el notebook original a la nueva qu estoy haciendo como si solo hiciera una copia

**Reply by Gracesl** [reply]
No tenis aue hacer mas notebooks, salvo par el inicio entorno de trabajo y la kata 10 manejo de errores y excepciones

**Comment by doguedogue** [comment, marked as answer]
Solo al crear un nuevo archivo y darle guardar como extensión .ipynb ya es un notebook.

puedes agregar carpetas para ser mas ordenado. 

![image](https://user-images.githubusercontent.com/23409026/153510301-daa0f2b3-a7cc-4b6e-911e-a04d607bd971.png)

![image](https://user-images.githubusercontent.com/23409026/153510556-458df2a2-c383-4c14-b9bf-17bac07cdc23.png)


No olvides cerrar la pregunta si consideras que fue contestada.
![image](https://user-images.githubusercontent.com/23409026/153510433-cb818a90-9b54-4616-b5d5-c02d200bcd8f.png)
